test/default-messages.js

```diff
@@ -1,37 +1,32 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath, [ __dirname + '/messages/defaults.js' ]);
```

Comment:
That gist uses slashes in `path.relative` (which also won't work, I think), but no, we'd want `path.join(__dirname, 'messages', 'defaults.js')` (you can verify because `path.join('a', 'b\\c')` doesn't result in `a/b/c`, so you can assume that the reverse wouldn't hold in Windows). Yes, it'd be great to update them all.